wicket/src/cli/inventory.rs

```diff
@@ -0,0 +1,110 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Support for inventory checks via wicketd.
+
+use crate::cli::CommandOutput;
+use crate::wicketd::create_wicketd_client;
+use anyhow::Context;
+use anyhow::Result;
+use clap::Subcommand;
+use owo_colors::OwoColorize;
+use sled_hardware_types::Baseboard;
+use slog::Logger;
+use std::net::SocketAddrV6;
+use std::time::Duration;
+use wicket_common::rack_setup::BootstrapSledDescription;
+
+const WICKETD_TIMEOUT: Duration = Duration::from_secs(5);
+
+#[derive(Debug, Subcommand)]
+pub(crate) enum InventoryArgs {
+    /// List state of all bootstrap sleds, as configured with rack-setup
+    ConfiguredBootstrapSleds {
+        /// Print output as json
+        #[clap(long)]
+        json: bool,
```

Comment:
suuuper nitpicky, doesn't actually matter: elsewhere, when writing CLI commands with multiple output formats, i like to define an output format enum and have a `--output plain` or `--output text` in addition to `--output json`/`--output csv` or what-have-you, rather than having a boolean flag for a specific format. that has two very small advantages IMO:

1. if adding an additional output format, it makes it a bit clearer in the CLI that the output formats are mutually exclusive,
2. when programmatically invoking the command, it's a bit nicer to just have to change the value of an arg based on the desired output format, rather than conditionally add or not-add an arg. e.g., a script invoking this can just always add an `--output` flag and pass `plain` or `json`, rather than adding/removing the `--json`.

but, OTTOH, this is a bit more code as it requires adding an enum and stuff...and, if other wicket commands have a `--json` already (AFAIK they don't, but i may have missed something), let's keep it consistent.

Reply:
Agreed about this! I'd use `--format json` and `--format human`.